### ISSUE TITLE
conda create -n test python=2.7 --offline  --no-update-dependencies --debug tried to access Internet

### ISSUE THREAD

**Issue by tarrade** [conda/conda#4699]
Hi there,

`conda create -n test python=2.7 --offline  --no-update-dependencies --debug`

to tried to reach anaconda website. Since I am using the --offline flag, I am nit sure why this append.

I just want to create an env with python 2.7 which is already in my base but it seems it find new packages to install. I am using Mac but I had similar issue on Linux. My aim is to create some env offline with an input list of packages (tar.bz2)

here the details:

```
$conda create -n test python=2.7 --offline  --no-update-dependencies --debug
DEBUG conda.gateways.disk.test:prefix_is_writable(57): testing write access for prefix '/Users/tarrade/anaconda' using path '/Users/tarrade/anaconda/conda-meta/history'
DEBUG conda.core.index:fetch_index(547): channel_urls=OrderedDict([(u'https://repo.continuum.io/pkgs/free/osx-64', (u'defaults', 0)), (u'https://repo.continuum.io/pkgs/free/noarch', (u'defaults', 0)), (u'https://repo.continuum.io/pkgs/r/osx-64', (u'defaults', 0)), (u'https://repo.continuum.io/pkgs/r/noarch', (u'defaults', 0)), (u'https://repo.continuum.io/pkgs/pro/osx-64', (u'defaults', 0)), (u'https://repo.continuum.io/pkgs/pro/noarch', (u'defaults', 0))])
Fetching package metadata ...DEBUG conda.core.index:fetch_repodata(481): Using cached repodata for https://repo.continuum.io/pkgs/free/osx-64 at /Users/tarrade/anaconda/pkgs/cache/d85a531e.json. Timeout in -42676 sec
DEBUG conda.core.index:read_pickled_repodata(376): found pickle file /Users/tarrade/anaconda/pkgs/cache/d85a531e.q
.DEBUG conda.core.index:fetch_repodata(481): Using cached repodata for https://repo.continuum.io/pkgs/free/noarch at /Users/tarrade/anaconda/pkgs/cache/9ca791dd.json. Timeout in -42678 sec
DEBUG conda.core.index:read_pickled_repodata(376): found pickle file /Users/tarrade/anaconda/pkgs/cache/9ca791dd.q
.DEBUG conda.core.index:fetch_repodata(481): Using cached repodata for https://repo.continuum.io/pkgs/r/osx-64 at /Users/tarrade/anaconda/pkgs/cache/0cdb2265.json. Timeout in -42678 sec
DEBUG conda.core.index:read_pickled_repodata(376): found pickle file /Users/tarrade/anaconda/pkgs/cache/0cdb2265.q
.DEBUG conda.core.index:fetch_repodata(481): Using cached repodata for https://repo.continuum.io/pkgs/r/noarch at /Users/tarrade/anaconda/pkgs/cache/716b88c0.json. Timeout in -42679 sec
.DEBUG conda.core.index:fetch_repodata(481): Using cached repodata for https://repo.continuum.io/pkgs/pro/osx-64 at /Users/tarrade/anaconda/pkgs/cache/abae4451.json. Timeout in -42679 sec
DEBUG conda.core.index:read_pickled_repodata(376): found pickle file /Users/tarrade/anaconda/pkgs/cache/abae4451.q
.DEBUG conda.core.index:fetch_repodata(481): Using cached repodata for https://repo.continuum.io/pkgs/pro/noarch at /Users/tarrade/anaconda/pkgs/cache/08159f17.json. Timeout in -42679 sec
.
DEBUG conda.gateways.disk.test:prefix_is_writable(57): testing write access for prefix '/Users/tarrade/anaconda' using path '/Users/tarrade/anaconda/conda-meta/history'
DEBUG conda.plan:add_defaults_to_specs(391): H0 specs=['python 2.7*']
DEBUG conda.plan:add_defaults_to_specs(401): H1 python
DEBUG conda.plan:add_defaults_to_specs(406): H2 lua False
DEBUG conda.plan:add_defaults_to_specs(411): H2A lua
DEBUG conda.plan:add_defaults_to_specs(437): HF specs=['python 2.7*']
DEBUG conda.plan:augment_specs(711): Pinned specs=[]
DEBUG conda.resolve:install_specs(794): Checking satisfiability of current install
DEBUG conda.resolve:bad_installed(744): Checking if the current environment is consistent
Solving package specifications: DEBUG conda.resolve:solve(849): Solving for [MatchSpec('python 2.7*')]
DEBUG conda.resolve:get_reduced_index(380): Retrieving packages for: [MatchSpec('python 2.7*')]
DEBUG conda.resolve:filter_group(405): python: pruned from 76 -> 32
DEBUG conda.resolve:filter_group(405): pip: pruned from 107 -> 32
.DEBUG conda.logic:minimize(485): Empty objective, trivial solution
DEBUG conda.resolve:solve(889): Package removal metric: 0
DEBUG conda.logic:minimize(557): Final peak objective: 0
DEBUG conda.logic:minimize(557): Final peak objective: 0
DEBUG conda.resolve:solve(895): Initial package version/build metrics: 0/0
DEBUG conda.logic:minimize(485): Empty objective, trivial solution
DEBUG conda.resolve:solve(900): Track feature count: 0
DEBUG conda.logic:minimize(557): Final sum objective: 1
DEBUG conda.logic:minimize(569): New peak objective: 1
DEBUG conda.resolve:solve(906): Package feature count: 0
DEBUG conda.logic:minimize(485): Empty objective, trivial solution
DEBUG conda.resolve:solve(911): Dependency update count: 0
DEBUG conda.logic:minimize(557): Final peak objective: 0
DEBUG conda.logic:minimize(557): Final peak objective: 0
DEBUG conda.resolve:solve(917): Additional package version/build metrics: 0/0
DEBUG conda.logic:minimize(557): Final sum objective: 8
DEBUG conda.logic:minimize(569): New peak objective: 1
DEBUG conda.resolve:solve(922): Weak dependency count: 8
DEBUG conda.resolve:solve(927): Looking for alternate solutions


Package plan for installation in environment /Users/tarrade/anaconda/envs/test:

The following NEW packages will be INSTALLED:

    openssl:    1.0.2k-0     
    pip:        9.0.1-py27_1 
    python:     2.7.13-0     
    readline:   6.2-2        
    setuptools: 27.2.0-py27_0
    sqlite:     3.13.0-0     
    tk:         8.5.18-0     
    wheel:      0.29.0-py27_0
    zlib:       1.2.8-3      

Proceed ([y]/n)? y

DEBUG conda.plan:plan_from_actions(340): Adding plans for operations: ('CHECK_FETCH', 'RM_FETCHED', 'FETCH', 'CHECK_EXTRACT', 'RM_EXTRACTED', 'EXTRACT', 'UNLINK', 'LINK', 'SYMLINK_CONDA')
DEBUG conda.plan:plan_from_actions(355): appending value defaults::openssl-1.0.2k-0 for action LINK
DEBUG conda.plan:plan_from_actions(355): appending value defaults::readline-6.2-2 for action LINK
DEBUG conda.plan:plan_from_actions(355): appending value defaults::sqlite-3.13.0-0 for action LINK
DEBUG conda.plan:plan_from_actions(355): appending value defaults::tk-8.5.18-0 for action LINK
DEBUG conda.plan:plan_from_actions(355): appending value defaults::zlib-1.2.8-3 for action LINK
DEBUG conda.plan:plan_from_actions(355): appending value defaults::python-2.7.13-0 for action LINK
DEBUG conda.plan:plan_from_actions(355): appending value defaults::setuptools-27.2.0-py27_0 for action LINK
DEBUG conda.plan:plan_from_actions(355): appending value defaults::wheel-0.29.0-py27_0 for action LINK
DEBUG conda.plan:plan_from_actions(355): appending value defaults::pip-9.0.1-py27_1 for action LINK
DEBUG conda.plan:plan_from_actions(355): appending value /Users/tarrade/anaconda for action SYMLINK_CONDA
DEBUG conda.core.package_cache:__init__(435): instantiating ProgressiveFetchExtract with
  defaults::openssl-1.0.2k-0
  defaults::readline-6.2-2
  defaults::sqlite-3.13.0-0
  defaults::tk-8.5.18-0
  defaults::zlib-1.2.8-3
  defaults::python-2.7.13-0
  defaults::setuptools-27.2.0-py27_0
  defaults::wheel-0.29.0-py27_0
  defaults::pip-9.0.1-py27_1

DEBUG conda.core.package_cache:is_writable(290): package cache directory '/Users/tarrade/.conda/pkgs' does not exist
DEBUG conda.core.package_cache:prepare(461): prepared package cache actions:
  cache_actions:
    CacheUrlAction<url=u'https://repo.continuum.io/pkgs/free/osx-64/openssl-1.0.2k-0.tar.bz2', target_full_path=u'/Users/tarrade/anaconda/pkgs/openssl-1.0.2k-0.tar.bz2'>
    CacheUrlAction<url=u'https://repo.continuum.io/pkgs/free/osx-64/setuptools-27.2.0-py27_0.tar.bz2', target_full_path=u'/Users/tarrade/anaconda/pkgs/setuptools-27.2.0-py27_0.tar.bz2'>
    CacheUrlAction<url=u'https://repo.continuum.io/pkgs/free/osx-64/wheel-0.29.0-py27_0.tar.bz2', target_full_path=u'/Users/tarrade/anaconda/pkgs/wheel-0.29.0-py27_0.tar.bz2'>
  extract_actions:
    ExtractPackageAction<source_full_path=u'/Users/tarrade/anaconda/pkgs/openssl-1.0.2k-0.tar.bz2', target_full_path=u'/Users/tarrade/anaconda/pkgs/openssl-1.0.2k-0'>
    ExtractPackageAction<source_full_path=u'/Users/tarrade/anaconda/pkgs/setuptools-27.2.0-py27_0.tar.bz2', target_full_path=u'/Users/tarrade/anaconda/pkgs/setuptools-27.2.0-py27_0'>
    ExtractPackageAction<source_full_path=u'/Users/tarrade/anaconda/pkgs/wheel-0.29.0-py27_0.tar.bz2', target_full_path=u'/Users/tarrade/anaconda/pkgs/wheel-0.29.0-py27_0'>

DEBUG conda.instructions:execute_instructions(243): executing plan [(u'PREFIX', u'/Users/tarrade/anaconda/envs/test'), ('PRINT', u'Linking packages ...'), ('PROGRESSIVEFETCHEXTRACT', <conda.core.package_cache.ProgressiveFetchExtract object at 0x1079d7090>), ('UNLINKLINKTRANSACTION', ((), (Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'openssl-1.0.2k-0', name=u'openssl', version=u'1.0.2k', build_string=u'0', build_number=0, with_features_depends=None, base_url=None, platform=None), Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'readline-6.2-2', name=u'readline', version=u'6.2', build_string=u'2', build_number=2, with_features_depends=None, base_url=None, platform=None), Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'sqlite-3.13.0-0', name=u'sqlite', version=u'3.13.0', build_string=u'0', build_number=0, with_features_depends=None, base_url=None, platform=None), Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'tk-8.5.18-0', name=u'tk', version=u'8.5.18', build_string=u'0', build_number=0, with_features_depends=None, base_url=None, platform=None), Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'zlib-1.2.8-3', name=u'zlib', version=u'1.2.8', build_string=u'3', build_number=3, with_features_depends=None, base_url=None, platform=None), Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'python-2.7.13-0', name=u'python', version=u'2.7.13', build_string=u'0', build_number=0, with_features_depends=None, base_url=None, platform=None), Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'setuptools-27.2.0-py27_0', name=u'setuptools', version=u'27.2.0', build_string=u'py27_0', build_number=0, with_features_depends=None, base_url=None, platform=None), Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'wheel-0.29.0-py27_0', name=u'wheel', version=u'0.29.0', build_string=u'py27_0', build_number=0, with_features_depends=None, base_url=None, platform=None), Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'pip-9.0.1-py27_1', name=u'pip', version=u'9.0.1', build_string=u'py27_1', build_number=1, with_features_depends=None, base_url=None, platform=None)))), ('LINK', Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'openssl-1.0.2k-0', name=u'openssl', version=u'1.0.2k', build_string=u'0', build_number=0, with_features_depends=None, base_url=None, platform=None)), ('LINK', Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'readline-6.2-2', name=u'readline', version=u'6.2', build_string=u'2', build_number=2, with_features_depends=None, base_url=None, platform=None)), ('LINK', Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'sqlite-3.13.0-0', name=u'sqlite', version=u'3.13.0', build_string=u'0', build_number=0, with_features_depends=None, base_url=None, platform=None)), ('LINK', Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'tk-8.5.18-0', name=u'tk', version=u'8.5.18', build_string=u'0', build_number=0, with_features_depends=None, base_url=None, platform=None)), ('LINK', Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'zlib-1.2.8-3', name=u'zlib', version=u'1.2.8', build_string=u'3', build_number=3, with_features_depends=None, base_url=None, platform=None)), ('LINK', Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'python-2.7.13-0', name=u'python', version=u'2.7.13', build_string=u'0', build_number=0, with_features_depends=None, base_url=None, platform=None)), ('LINK', Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'setuptools-27.2.0-py27_0', name=u'setuptools', version=u'27.2.0', build_string=u'py27_0', build_number=0, with_features_depends=None, base_url=None, platform=None)), ('LINK', Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'wheel-0.29.0-py27_0', name=u'wheel', version=u'0.29.0', build_string=u'py27_0', build_number=0, with_features_depends=None, base_url=None, platform=None)), ('LINK', Dist(_Dist__initd=True, channel=u'defaults', dist_name=u'pip-9.0.1-py27_1', name=u'pip', version=u'9.0.1', build_string=u'py27_1', build_number=1, with_features_depends=None, base_url=None, platform=None)), ('SYMLINK_CONDA', '/Users/tarrade/anaconda')]
DEBUG conda.instructions:execute_instructions(249):  PREFIX(u'/Users/tarrade/anaconda/envs/test')
DEBUG conda.instructions:execute_instructions(249):  PRINT(u'Linking packages ...')
DEBUG conda.instructions:execute_instructions(249):  PROGRESSIVEFETCHEXTRACT(<conda.core.package_cache.ProgressiveFetchExtract object at 0x1079d7090>)
DEBUG conda.core.package_cache:_execute_action(483): Error in action CacheUrlAction<url=u'https://repo.continuum.io/pkgs/free/osx-64/openssl-1.0.2k-0.tar.bz2', target_full_path=u'/Users/tarrade/anaconda/pkgs/openssl-1.0.2k-0.tar.bz2'>
DEBUG conda.core.package_cache:_execute_action(484): Traceback (most recent call last):
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/core/package_cache.py", line 481, in _execute_action
    action.execute()
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/core/path_actions.py", line 749, in execute
    download(self.url, self.target_full_path, self.md5sum)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/gateways/download.py", line 63, in download
    resp = session.get(url, stream=True, proxies=session.proxies, timeout=timeout)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 501, in get
    return self.request('GET', url, **kwargs)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 488, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 609, in send
    r = adapter.send(request, **kwargs)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/connection.py", line 70, in send
    raise RuntimeError(message)
RuntimeError: EnforceUnusedAdapter called with url https://repo.continuum.io/pkgs/free/osx-64/openssl-1.0.2k-0.tar.bz2
This command is using a remote connection in offline mode.


DEBUG conda.core.package_cache:_execute_action(483): Error in action CacheUrlAction<url=u'https://repo.continuum.io/pkgs/free/osx-64/openssl-1.0.2k-0.tar.bz2', target_full_path=u'/Users/tarrade/anaconda/pkgs/openssl-1.0.2k-0.tar.bz2'>
DEBUG conda.core.package_cache:_execute_action(484): Traceback (most recent call last):
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/core/package_cache.py", line 481, in _execute_action
    action.execute()
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/core/path_actions.py", line 749, in execute
    download(self.url, self.target_full_path, self.md5sum)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/gateways/download.py", line 63, in download
    resp = session.get(url, stream=True, proxies=session.proxies, timeout=timeout)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 501, in get
    return self.request('GET', url, **kwargs)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 488, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 609, in send
    r = adapter.send(request, **kwargs)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/connection.py", line 70, in send
    raise RuntimeError(message)
RuntimeError: EnforceUnusedAdapter called with url https://repo.continuum.io/pkgs/free/osx-64/openssl-1.0.2k-0.tar.bz2
This command is using a remote connection in offline mode.


DEBUG conda.core.package_cache:_execute_action(483): Error in action CacheUrlAction<url=u'https://repo.continuum.io/pkgs/free/osx-64/openssl-1.0.2k-0.tar.bz2', target_full_path=u'/Users/tarrade/anaconda/pkgs/openssl-1.0.2k-0.tar.bz2'>
DEBUG conda.core.package_cache:_execute_action(484): Traceback (most recent call last):
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/core/package_cache.py", line 481, in _execute_action
    action.execute()
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/core/path_actions.py", line 749, in execute
    download(self.url, self.target_full_path, self.md5sum)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/gateways/download.py", line 63, in download
    resp = session.get(url, stream=True, proxies=session.proxies, timeout=timeout)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 501, in get
    return self.request('GET', url, **kwargs)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 488, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 609, in send
    r = adapter.send(request, **kwargs)
  File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/connection.py", line 70, in send
    raise RuntimeError(message)
RuntimeError: EnforceUnusedAdapter called with url https://repo.continuum.io/pkgs/free/osx-64/openssl-1.0.2k-0.tar.bz2
This command is using a remote connection in offline mode.


An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues



INFO conda.common.io:captured(59): overtaking stderr and stdout
DEBUG conda.gateways.disk.test:prefix_is_writable(57): testing write access for prefix '/Users/tarrade/anaconda' using path '/Users/tarrade/anaconda/conda-meta/history'
DEBUG conda.gateways.disk.test:prefix_is_writable(57): testing write access for prefix '/Users/tarrade/anaconda' using path '/Users/tarrade/anaconda/conda-meta/history'
INFO conda.common.io:captured(65): stderr and stdout yielding back
Current conda install:

               platform : osx-64
          conda version : 4.3.11
       conda is private : False
      conda-env version : 4.3.11
    conda-build version : 2.0.12
         python version : 2.7.13.final.0
       requests version : 2.12.4
       root environment : /Users/tarrade/anaconda  (writable)
    default environment : /Users/tarrade/anaconda
       envs directories : /Users/tarrade/anaconda/envs
                          /Users/tarrade/.conda/envs
          package cache : /Users/tarrade/anaconda/pkgs
                          /Users/tarrade/.conda/pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/osx-64  (offline)
                          https://repo.continuum.io/pkgs/free/noarch  (offline)
                          https://repo.continuum.io/pkgs/r/osx-64  (offline)
                          https://repo.continuum.io/pkgs/r/noarch  (offline)
                          https://repo.continuum.io/pkgs/pro/osx-64  (offline)
                          https://repo.continuum.io/pkgs/pro/noarch  (offline)
            config file : /Users/tarrade/.condarc
           offline mode : True
             user-agent : conda/4.3.11 requests/2.12.4 CPython/2.7.13 Darwin/15.6.0 OSX/10.11.6
                UID:GID : 501:20



`$ /Users/tarrade/anaconda/bin/conda create -n test python=2.7 --offline --no-update-dependencies --debug`




    Traceback (most recent call last):
      File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/exceptions.py", line 616, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 137, in _main
        exit_code = args.func(args, p)
      File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/cli/main_create.py", line 68, in execute
        install(args, parser, 'create')
      File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/cli/install.py", line 359, in install
        execute_actions(actions, index, verbose=not context.quiet)
      File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/plan.py", line 825, in execute_actions
        execute_instructions(plan, index, verbose)
      File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/instructions.py", line 258, in execute_instructions
        cmd(state, arg)
      File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/instructions.py", line 111, in PROGRESSIVEFETCHEXTRACT_CMD
        progressive_fetch_extract.execute()
      File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/core/package_cache.py", line 470, in execute
        self._execute_action(action)
      File "/Users/tarrade/anaconda/lib/python2.7/site-packages/conda/core/package_cache.py", line 492, in _execute_action
        raise CondaMultiError(exceptions)
    CondaMultiError: RuntimeError(u'EnforceUnusedAdapter called with url https://repo.continuum.io/pkgs/free/osx-64/openssl-1.0.2k-0.tar.bz2\nThis command is using a remote connection in offline mode.\n',)
    RuntimeError(u'EnforceUnusedAdapter called with url https://repo.continuum.io/pkgs/free/osx-64/openssl-1.0.2k-0.tar.bz2\nThis command is using a remote connection in offline mode.\n',)
    RuntimeError(u'EnforceUnusedAdapter called with url https://repo.continuum.io/pkgs/free/osx-64/openssl-1.0.2k-0.tar.bz2\nThis command is using a remote connection in offline mode.\n',)
   
``` 

```
$ conda info
Current conda install:

               platform : osx-64
          conda version : 4.3.11
       conda is private : False
      conda-env version : 4.3.11
    conda-build version : 2.0.12
         python version : 2.7.13.final.0
       requests version : 2.12.4
       root environment : /Users/tarrade/anaconda  (writable)
    default environment : /Users/tarrade/anaconda
       envs directories : /Users/tarrade/anaconda/envs
                          /Users/tarrade/.conda/envs
          package cache : /Users/tarrade/anaconda/pkgs
                          /Users/tarrade/.conda/pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/osx-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/osx-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/osx-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : /Users/tarrade/.condarc
           offline mode : False
             user-agent : conda/4.3.11 requests/2.12.4 CPython/2.7.13 Darwin/15.6.0 OSX/10.11.6
                UID:GID : 501:20

```

**Comment by kalefranz** [comment]
If you have a specific list of tarballs, 

    conda create -n test ./path/to/package1.tar.bz2 ./path/to/package1.tar.bz2

should work, or create a file

```
@EXPLICIT
file:///path/to/package1.tar.bz2
file:///path/to/package2.tar.bz2
```

named, say, `packages.txt`, and use

    conda create -n test --file packages.txt

For the contents of the file, you might end up not needing the `file://` part; I don't remember.

**Comment by kalefranz** [comment]
When using the `--offline` flag, all the necessary data already has to be in the package caches.  Otherwise we have problems.  We shouldn't be raising `CondaRuntimeError` there though.

CC @tyler-thetyrant we can discuss how to fix this tomorrow, since it touches on what you're doing now.

**Comment by tarrade** [comment]
Hi @kalefranz ,

Thanks, I made to made a script to extract the list of packages from Windows and download the Linux version. It is a little bit tricky with the versioning of the packages but it works.

I tried your command:
`conda create -n test ./path/to/package1.tar.bz2 ./path/to/package1.tar.bz2`

and it seems to works but it display no messages at all! I don't need a debug or verbose messages but only that it say what was working and what didn't work. I git no messages but I have some missing lib. For example 
> rpy2
need some other lib like 
> libpcre.so.1
Why it is not checked during the installation ? It is not possible to have this without Internet connection ? 

Thanks
Cheers
Fabien
